### PR TITLE
Portfolio details popup window

### PR DESCRIPTION
**Pull request by Yulianav09**
### Portfolio details popup window

- Added a popup window for projects cards
- Cards are generated dynamically 
- popup window shows the information of the selected project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,30 @@
...
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+        </ul>
+        <div id="modalSnapshot"><img id="snapshoot" src="./src/Snapshoot Portfolio.png" alt="Snapshoot image"></div>
+          <div id="modalContent">
+            <p id="modalDescription" class="cardDescription textBlack">A daily selection of privately personalized reads; no accounts or sign-ups required.
+             has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <a target="_blank" id="liveDemo" href=""><button class="projectButton icons"><span>See live</span><img src="./src/Icon - Export.svg" alt=""></button></a>
+            <a target="_blank" id="source" href=""><button class="projectButton icons"><span>See source</span><img src="./src/Icon -GitHub.svg" alt=""></button></a>
```
Comment (shahadat3669) [inline index.html:31]: - [x] Well! Your button looks good. 👍 Currently, the gap between buttons is "12px", but as per the design, there should be a "24px" gap. Please consider updating the code to increase the gap between the buttons to match the design specifications. 🙏 
- [x] Additionally, it is not necessary to wrap the button tag with the "a" tag. You should choose either the "a" tag or the button tag, but using both is not necessary. Please update the code accordingly to ensure consistency and simplify the markup. 🙏

By making these adjustments, you will ensure better adherence to the design and improve the overall user experience.  Good luck with the updates! 💪🔧🎨

File: index.html [hunk 2]
```diff
@@ -9,6 +9,30 @@
...
+          </div>
+      </article>
```
Comment (shahadat3669) [inline index.html:33]: - [x] Well! Your model looks good. 👍 Currently, the padding is  "16px", but per the design, there should be a "24px". Please consider updating the code to increase the padding to match the design specifications. 🙏

File: index.html [hunk 3]
```diff
@@ -9,6 +9,30 @@
...
+      <article class="modal">
+        <div id="modalHeading">
+          <h3 class="modaltitle">Keeping track of hundreds  of components website</h3>
+          <div id="close" class="closex"><img src="./src/Icon - Cancel.svg" alt="close modal window"></div>
+        </div>
+        <ul id="modalTecnologies" class="boxes">
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+        </ul>
```
Comment (shahadat3669) [inline index.html:24]: - [x] Well! Your tags' color and font styles look good. 👍 Currently, the gap between tags is "12px", but as per the design, there should be an "8px" gap. Please consider updating the code to reduce the gap between the tags to match the design specifications. 🙏💻

By making this adjustment, you will ensure better alignment and spacing of the tags, creating a more visually pleasing layout that aligns with the design guidelines. ✨🚀

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi Everyone, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✅ Good PR Title
✅ Good PR Description
✅ Good use of GitHub Flows

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Have any doubt ❓


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## ⚠️ WARNING ⚠️
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Yulianav09** [comment]
Hello @shahadat3669, thanks for the feedback, all the changes were applied

**Pull request merged**